npm-shrinkwrap.json

```diff
@@ -101,6 +101,23 @@
       "from": "any-promise@>=1.1.0 <2.0.0",
       "resolved": "https://registry.npmjs.org/any-promise/-/any-promise-1.3.0.tgz"
     },
+    "appdmg": {
```

Comment:
Did you run `npm run shrinkwrap` after installing the dependencies? That script is exactly meant to remove these kind of things.